product_variant_multi_link/models/product_product.py

```diff
@@ -14,6 +15,46 @@ class ProductProduct(models.Model):
         compute="_compute_product_link_ids",
     )
 
+    product_product_link_count = fields.Integer(
+        string="Product Links Count", compute="_compute_product_product_link_count"
```

Comment:
Runbot is complaining about this label:

    Two fields (product_template_link_count, product_product_link_count) of product.product() have the same label: Product Links Count.

Can you handle it? Maybe "Variant Links Count"? (I don't know the context)

Reply:
should be fine now.